volume/sync.py

```diff
@@ -88,6 +88,20 @@ def main():
     print("Synchronising..")
     for project in projects:
         if project["name"] in existing_projects:
+            # Update task types
+            existing_project = existing_projects[project["name"]]
+            existing_project_task_types = existing_project["config"]["tasks"]
+            if existing_project_task_types != tasks:
+                print(
+                    "Updating tasks types on \"{0}\" to:\n{1}".format(
+                        project["name"], tasks
+                    )
+                )
+                existing_project["config"]["tasks"] = tasks
+                client = pymongo.MongoClient(os.environ["AVALON_MONGO"])
+                db = client["avalon"]
+                db[project["name"]].save(existing_project)
```

Comment:
Save is deprecated.

http://api.mongodb.com/python/current/api/pymongo/collection.html#pymongo.collection.Collection.save

Also, if we are using pymongo directly, we could use *only* pymongo and discard `io.py`.